backend/api/simulation.py

```
@@ -364,11 +431,20 @@ async def start_simulation(
             }
             for persona in all_personas
         ]
     }
+    # Debug which personas have custom system prompts at start
+    try:
+        for p in scenario_data.get("personas", []):
+            name = p.get("identity", {}).get("name")
+            has_custom = bool(p.get("system_prompt"))
+            print(f"[START] Persona loaded: name={name}, has_custom_prompt={has_custom}")
+    except Exception:
+        pass
```

Ruff (0.14.0)

`try`-`except`-`pass` detected, consider logging the exception

(S110)